packages/sdk/ts/tally/results.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  MACI__factory as MACIFactory,
+  Tally__factory as TallyFactory,
+  Poll__factory as PollFactory,
+} from "@maci-protocol/contracts/typechain-types";
+
+import type { IGetResultPerOptionArgs, IGetResultsArgs } from "./types";
+
+/**
+ * Get result per option
+ * @param param0 - The get result per option args
```

Comment:
We could use the type name here in the docs like 
```javascript
/*
* @param {IGetResultPerOptionsArgs} - The arguments to get result per option
*/
```

We have something similar in multiple functions of the sdk. E.g.

1.  [`/packages/sdk/signup.ts` -> `getSignedupUserData`](https://github.com/privacy-scaling-explorations/maci/blob/85bba2dfa28f41c8653168f2283b99d2ca353128/packages/sdk/ts/user/signup.ts#L13)
2. https://github.com/privacy-scaling-explorations/maci/blob/85bba2dfa28f41c8653168f2283b99d2ca353128/packages/sdk/ts/user/joinPoll.ts#L16